<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSavePlot</class>
 <widget class="QWidget" name="DialogSavePlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save chart</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNameOpen">
       <property name="toolTip">
        <string>Choose file name</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ssdb.qrc">
         <normaloff>:/img/file.png</normaloff>:/img/file.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelPath">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPathOpen">
       <property name="toolTip">
        <string>Choose directory path</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ssdb.qrc">
         <normaloff>:/img/open_path.png</normaloff>:/img/open_path.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelFormat">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFormat">
       <property name="toolTip">
        <string>Choose file format</string>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="Russian" country="Ukraine"/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ssdb.qrc"/>
 </resources>
 <connections/>
</ui>
